src/Main.java

```diff
@@ -1,3 +1,8 @@
+import interfaceticket.Sender;
+import sending.SendByEmail;
+import sending.SendByPhone;
+import userANDticket.*;
```

Comment:
Comment related all package naming.
Avoid ‘AND’ in your naming, if you would like to name with And than it’s good point to have separate packages/classes/functions
Also seems like TicketService is not quite fit in this package, we usually group by functionality, not by model(but if it's so, then we need to keep this principle through the whole project :)

Soooo, lets rename userANDticket to 'model', where Admin, User, Ticket and Client will be stored
Than 'exception' leave as is
Add new 'service' package where TicketService will be 
Inside 'interface' package create separate packages inside, let it be 'sending' where SendByPhone/Email + Sender, Shareable is located, all other just inside interface